run_configure.mk

```diff
@@ -47,6 +47,10 @@ help:
 # Remove all built-in rules
 .SUFFIXES:
 
+
+# Set a default value for CONFIG_SHELL
+CONFIG_SHELL ?= sh
```

Comment:
Even though it used to use `sh`, perhaps the default should be `bash`?

Reply:
there is no guarantee that bash will be installed on the machine (or on the path), but posix requires that `sh` exists

Reply:
We already assume bash is available:

https://github.com/eclipse/omr/blob/916cfcfa682c496a7f8a5a1fb611fb7d0a2d8c0c/ddr_artifacts.mk#L36
https://github.com/eclipse/omr/blob/916cfcfa682c496a7f8a5a1fb611fb7d0a2d8c0c/fvtest/omrtest.mk#L48

Reply:
That is only DDR. I would argue that we should really try to keep things posix compliant where possible. (Not to mention the risk of breaking builds on some of the more o
bscure systems)

Reply:
is bash required by autotools (configure)?

Reply:
It is not. They go to great lengths to work oon just about any posix-ish shell. (although I imagine bash is preferred and gets the most testing)

Reply:
The only reason we're having this discussion, though, is because some platforms have `sh` that *isn't* good enough (e.g. Ubuntu 16.04 on x86 or 390x).

Reply:
only in a somewhat weird edge case. Also, I need to look into it, but i believe that may actually just be a bug in dash

Reply:
Ubuntu 18.04 on x86 has the same behavior as 16.04.

Reply:
right but as far as I can tell, this should work according to the POSIX spec, thus it is logically a bug in the shell